fhir-bulkimportexport-webapp/src/main/java/com/ibm/fhir/bulkimport/ChunkWriter.java

```diff
@@ -188,12 +189,18 @@ public void writeItems(List<java.lang.Object> arg0) throws Exception {
 
             for (Resource fhirResource : fhirResourceList) {
                 try {
+                    String id = fhirResource.getId();
+                    if (id == null) {
+                        id = UUID.randomUUID().toString();
+                    }
+
```

Comment:
not sure if assigning a random id to resource with null id is the right approach.  because assigning random id doesn't help to preserver the relation between resources. so I chose to regard those resource with empty id as invalid resources which will be handled by the exception handler codes automatically.

Reply:
You have confirmed my thoughts.  I have removed and added a comment.